sdk/go/common/resource/plugin/langruntime.go

```diff
@@ -270,3 +277,44 @@ func UnmarshallRuntimeOptionPrompt(p *pulumirpc.RuntimeOptionPrompt) (RuntimeOpt
 		PromptType:  RuntimeOptionType(p.PromptType),
 	}, nil
 }
+
+// MakeRuntimeOptionPromptChoices creates a list of runtime option values from a list of executable names.
+// If an executable is not found, it will be listed with a `[not found]` suffix at the end of the list.
+func MakeExecutablePromptChoices(executables ...string) []*pulumirpc.RuntimeOptionPrompt_RuntimeOptionValue {
+	type packagemanagers struct {
+		name  string
+		found bool
+	}
+	pms := []packagemanagers{}
+	for _, pm := range executables {
+		found := false
+		if _, err := exec.LookPath(pm); err != nil {
+			found = true
+		}
+		pms = append(pms, packagemanagers{pm, found})
+	}
+
+	sort.SliceStable(pms, func(i, j int) bool {
+		// Don't reorder if both are found or both are not found.
+		if pms[i].found == pms[j].found {
+			return false
+		}
+		// pms[i] is less than pms[j] if pms[i] is not found.
+		return !pms[i].found
+	})
+
+	choices := []*pulumirpc.RuntimeOptionPrompt_RuntimeOptionValue{}
+	caser := cases.Title(language.English)
+	for _, pm := range pms {
+		displayName := caser.String(pm.name)
```

Comment:
Why are we casing this? Does "Npm" really look better than just "npm" like it was before?